pkg/testcase/restart.go

```diff
@@ -0,0 +1,21 @@
+package testcase
+
+import (
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestRestartService() {
+	product, err := shared.GetProduct()
+	Expect(err).NotTo(HaveOccurred(), "failed to get product")
+	c := factory.ClusterConfig(GinkgoT())
+
+	_, err = shared.ManageService(product, "restart", "server", c.ServerIPs)
+	Expect(err).NotTo(HaveOccurred(), "failed to get systemctl command")
```

Comment:
here the error message should be: failed to restart service. 
we do not know if the problem was getting sysctl command or not.

Reply:
the err if happens will return already , no need to log